src/components/Game.js

```diff
@@ -37,6 +38,25 @@ function Game() {
   const [currentIndex,setCurrentIndex] = useState(0);
   // const [newtextresult, setnewtextresult] = useState('');
   // const [voiceTextHighlight, setVoiceTextHighLight] = useState('');
+
+  const [Player1,setPlayer1] = useState('');
+  const [Player2, setPlayer2]=useState('')
+
+  useEffect(()=>{
+
+    const token = localStorage.getItem('token');
```

Comment:
the same logic is used in avatar.js. Can we use the common logic?

Reply:
done it. i created another hellper file for this